boards/dactyl_manuform/readme.md

```diff
@@ -0,0 +1,49 @@
+# Dactyl Manuform
+
+The [Dactyl Manuform](https://github.com/tshort/dactyl-keyboard) is a handwired, split bodied, tented, curved keywell keyboard - adapted from the [Dactyl](https://github.com/adereth/dactyl-keyboard) keyboard with the thumb cluster design of the [ManuForm](https://geekhack.org/index.php?topic=46015.0) keyboard. Information for building a Dactyl Manuform found in first link.
```

Comment:
Break up long lines to preferable 80 characters.
External links should have the current date attached (i.e.: last known valid date), if there's the potential that the link may change withing a couple of years. (Yes, that caused issues in the past3)

Reply:
Hi @xs5871,

looking at other readme.md files I'm not seeing a date attached to external links. Could you give example of how date should be added?

Reply:
Yeah, sadly we never finished the doc linting and doc guidlines.
"[text (YYYY-MM)](link)" should do. Not strictly necessary for expected long-lived external links; github should be fine without, unless it's a small personal project. I wasn't familiar with geekhack, but they seem to be very long lived as well, although that could benefit from a date of post creation.
The imgur could state roughly when the image was taken, for example.

Reply:
date added for ManuForm link